src/Layout/Footer/footer.styled.js

```diff
@@ -9,11 +9,30 @@ const FooterWrapper = styled.div`
   color: white;
   background-color: #1d1d35;
   z-index: 2;
+  height: 64px;
+`;
+
+const FooterItems = styled.div`
   display: flex;
   flex-direction: row;
-  justify-content: flex-end;
+  justify-content: space-between;
   align-items: center;
-  height: 64px;
+  width: 100%;
+`;
+
+const AddToLikedSongs = styled.div`
+  font-size: 20px;
+  margin-left: 16px;
+  svg {
+    path {
+      fill: ${props => (props.isALikedSong ? 'white' : 'transparent')};
+      stroke: white;
+      stroke-width: 35px;
+    }
+  }
+  &:hover {
+    font-size: 22px;
```

Comment:
I've tested this in my local and I think we should use `transform: scale(1.1)` instead of incrementing the font-size. It feels more natural. What do you think? Also, put a `cursor: pointer` to `AddToLikedSongs`.